docs/user/openstack/deploy_baremetal_workers.md

```diff
@@ -0,0 +1,188 @@
+# Deploying OpenShift bare-metal workers on OpenStack cloud provider
+TBD introduction
+
+## Table of Contents
+- [Prerequisites](#prerequisites)
+- [Conciderations whe deploying bare-metal workers](#conciderations-whe-deploying-bare-metal-workers)
+- [Deploying cluster with BM workers on tenant network deployed by the installer](#deploying-cluster-with-bm-workers-on-tenant-network-deployed-by-the-installer)
+- [Deploying cluster with BM workers on preexisting network](#deploying-cluster-with-bm-workers-on-preexisting-network)
+- [Deploy cluster with vm workers only and add BM workers after](#deploy-cluster-with-vm-workers-only-and-add-bm-workers-after)
+
+
+## Prerequisites
+* [Ironic bare metal service][2] is enabled an accessible through the [OpenStack Compute API][3].
+* Bare-metal workers are available as an [OpenStack flavor][1]
+* The image used to boot the bare-metal worker is available from the [Glance OpenStack image service][4] 
+* The cloud provider is configured such that bare-metal and vms can be attached to the same tenant network.
+* The cloud provider is configured such that vms can be deployed onto a preconfigured network which already has bare-metal servers attached to it.
+* [Ironic bare metal service][2] can listen for and PXE-boot nodes in tenant networks
+
+## Conciderations whe deploying bare-metal workers
+
+* Long boot times for bare-metal machines may result in deployment timeout
+
+    The first time a bare-metal server is booted the boot time could be significantly longer that
+    the first time a vm server is booted. This longer boot time could exceed the timeout settings of 
+    the installer. If that occurs, the deployment will fail with a timeout error.  The deployment maybe
+    restarted and completed by re-running  the installer with the appropriate *wait-for*
+    command. For example:
+        
+        ./openshift-install wait-for bootstrap-complete --log-level debug 
```

Comment:
I think you meant `install-complete` here and not `bootastrap-complete` (the Control Plane should be on VMs)